### ISSUE TITLE
Panel issue on 48b10

### ISSUE THREAD

**Issue by share3141** [Quicksaver/OmniSidebar#126]
When I click on the option button the only that opens is a blank screen and the preferences drop down menu at the bottom (undo, redo etc) none of which work.  My bookmarks open OK - I just can't get to any of the options.  I've attached an image
![screenshot - 7_25_2016 12_36_16 pm](https://cloud.githubusercontent.com/assets/20307002/17110835/78396b26-5264-11e6-8042-5a420c519251.png)

**Comment by Quicksaver** [comment]
Are you using the latest version of the add-on? Does that also happen if you disable all your other add-ons (including CCK2 if you use/have it) except for OSB?

**Comment by share3141** [comment]
I am using the latest version of the addon.  I did as you suggested and 
played around with enabling / disabling extensions.  "No Resource Leak 
1.0" appears to be the culprit.  With that being said while I was 
enabling / disabling extensions that particular extension was updatedto 
1.1.0.  The problem appears to have been solved.  I'll keep an eye on it 
and let you know if anything else crops up.
